utility/src/main/java/org/oppia/android/util/logging/EventBundleCreator.kt

```diff
@@ -23,6 +27,31 @@ const val SUB_TOPIC_ID_KEY = "subTopicId"
 const val QUESTION_ID_KEY = "questionId"
 const val EXPLORATION_ID_KEY = "explorationId"
 const val PRIORITY_KEY = "priority"
+const val LEARNER_ID_KEY_LEARNER_ANALYTICS = "profile_id"
```

Comment:
Move to #4248?

One approach might be to have one PR be just domain utility changes, one that introduces the events, and one that turns it on (and subsequently updates modules).